### PR TITLE
Seigaiha

### PR DESCRIPTION
**Pull request by yiancar**
Add Seigaiha keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by yiancar** [comment]
Bump Bump:)

File: keyboards/seigaiha/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# The default keymap for Seigaiha with VIA disabled
+
+![Layer 0](https://i.imgur.com/wO8lfpi.png)
+
+![Layer 1](https://i.imgur.com/4Q0Rk2V.png)
```
Comment (fauxpark) [inline keyboards/seigaiha/keymaps/default/readme.md:5]:
```suggestion
![Layer 0](https://i.imgur.com/wO8lfpih.png)

![Layer 1](https://i.imgur.com/4Q0Rk2Vh.png)
```

File: keyboards/seigaiha/keymaps/via/readme.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# The default keymap for Seigaiha with VIA enabled
+
+![Layer 0](https://i.imgur.com/wO8lfpi.png)
+
+![Layer 1](https://i.imgur.com/4Q0Rk2V.png)
```
Comment (fauxpark) [inline keyboards/seigaiha/keymaps/via/readme.md:5]:
```suggestion
![Layer 0](https://i.imgur.com/wO8lfpih.png)

![Layer 1](https://i.imgur.com/4Q0Rk2Vh.png)
```

File: keyboards/seigaiha/readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Seigaiha
+
+![Seigaiha](https://i.imgur.com/GCGyOmp.jpg?1)
```
Comment (fauxpark) [inline keyboards/seigaiha/readme.md:3]:
```suggestion
![Seigaiha](https://i.imgur.com/GCGyOmph.jpg)
```

File: keyboards/seigaiha/info.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (fauxpark) [inline keyboards/seigaiha/info.json:1]: Needs to be redone without the rotation.
image

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/seigaiha/info.json (hunk 4 above)
Comment (yiancar) [reply keyboards/seigaiha/info.json:1]: gotcha! on it:)

Comment (yiancar) [reply keyboards/seigaiha/info.json:1]: Done @fauxpark !

File: keyboards/seigaiha/info.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                {"label":"", "x":14.5, "y":3},
+                {"label":"", "x":15.5, "y":3, "w":1.75},
+                {"label":"", "x":17.25, "y":3},
+                {"label":"", "x":1.25, "y":4, "w":1.5},
+                {"label":"", "x":4.25, "y":4, "w":1.5},
+                {"label":"", "x":5.75, "y":4, "w":2},
+                {"label":"", "x":7.75, "y":4, "w":1.25},
+                {"label":"", "x":9.5, "y":4, "w":2.75},
+                {"label":"", "x":12.25, "y":4, "w":1.5},
+                {"label":"", "x":16.75, "y":4, "w":1.5}
```
Comment (fauxpark) [inline keyboards/seigaiha/info.json:75]:
```suggestion
    "width": 18.25,
    "height": 5,
    "layouts": {
        "LAYOUT_alice_split_bs": {
            "layout": [
                {"x":0, "y":0},
                {"x":1.25, "y":0},
                {"x":2.25, "y":0},
                {"x":3.25, "y":0},
                {"x":4.25, "y":0},
                {"x":5.25, "y":0},
                {"x":6.25, "y":0},
                {"x":7.25, "y":0},
                {"x":10.25, "y":0},
                {"x":11.25, "y":0},
                {"x":12.25, "y":0},
                {"x":13.25, "y":0},
                {"x":14.25, "y":0},
                {"x":15.25, "y":0},
                {"x":16.25, "y":0},
                {"x":17.25, "y":0},

                {"x":0, "y":1},
                {"x":1.25, "y":1, "w":1.5},
                {"x":2.75, "y":1},
                {"x":3.75, "y":1},
                {"x":4.75, "y":1},
                {"x":5.75, "y":1},
                {"x":6.75, "y":1},
                {"x":9.75, "y":1},
                {"x":10.75, "y":1},
                {"x":11.75, "y":1},
                {"x":12.75, "y":1},
                {"x":13.75, "y":1},
                {"x":14.75, "y":1},
                {"x":15.75, "y":1},
                {"x":16.75, "y":1, "w":1.5},

                {"x":0, "y":2},
                {"x":1.25, "y":2, "w":1.75},
                {"x":3, "y":2},
                {"x":4, "y":2},
                {"x":5, "y":2},
                {"x":6, "y":2},
                {"x":7, "y":2},
                {"x":10, "y":2},
                {"x":11, "y":2},
                {"x":12, "y":2},
                {"x":13, "y":2},
                {"x":14, "y":2},
                {"x":15, "y":2},
                {"x":16, "y":2, "w":2.25},

                {"x":1.25, "y":3, "w":2.25},
                {"x":3.5, "y":3},
                {"x":4.5, "y":3},
                {"x":5.5, "y":3},
                {"x":6.5, "y":3},
                {"x":7.5, "y":3},
                {"x":9.5, "y":3},
                {"x":10.5, "y":3},
                {"x":11.5, "y":3},
                {"x":12.5, "y":3},
                {"x":13.5, "y":3},
                {"x":14.5, "y":3},
                {"x":15.5, "y":3, "w":1.75},
                {"x":17.25, "y":3},

                {"x":1.25, "y":4, "w":1.5},
                {"x":4.25, "y":4, "w":1.5},
                {"x":5.75, "y":4, "w":2},
                {"x":7.75, "y":4, "w":1.25},
                {"x":9.5, "y":4, "w":2.75},
                {"x":12.25, "y":4, "w":1.5},
                {"x":16.75, "y":4, "w":1.5}
```
Just a bit of cleanup, fixing the width/height, removing empty labels.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**